test/browser.js

```diff
@@ -173,18 +173,20 @@ test('throws if does not support ReadableStream', withPage, async (t, page) => {
 });
 
 test('FormData with searchParams', withPage, async (t, page) => {
-	t.plan(2);
+	t.plan(3);
 
 	const server = await createTestServer();
 	server.get('/', (request, response) => {
 		response.end();
 	});
 	server.post('/', async (request, response) => {
 		const requestBody = await pBody(request);
+		const contentType = request.headers['content-type'];
+		const boundary = contentType.split('boundary=')[1];
 
+		t.regex(requestBody, new RegExp(boundary));
```

Comment:
I don't think it's wise to dynamically construct a regular expression for use in a test assertion. If the `boundary` happens to include some regex special characters, it could cause the test to produce unexpected results.

I feel that `requestBody.includes(boundary)` would be better.